cfme/services/myservice/ssui.py

```diff
@@ -165,6 +165,7 @@ def set_ownership(self, owner, group):
                'select_group': group})
     view.save_button.click()
     view = self.create_view(DetailsMyServiceView)
+    time.sleep(15)  # WA for StaleElementException in 5.8 testing
```

Comment:
Required:
@jan-zmeskal: I think you can add some wait to display this view like `view.wait_displayed("15s")`. I know first, you have to check `is_displayed` method for this view.